tests/screenshot/apps/components/Spinner.js

```diff
@@ -0,0 +1,11 @@
+import Spinner from '../../../../Spinner';
+import React from 'react';
+
+const SpinnerTests = [
+	<Spinner />,
+	<Spinner centered />,
```

Comment:
It display blank. May be we need to add parent div.
Please check the result image